databricks/koalas/series.py

```diff
@@ -317,8 +317,61 @@ def unique(self):
         # Pandas wants a series/array-like object
         return _col(self.to_dataframe().unique())
 
-    @derived_from(pd.Series)
+    # TODO: Update Documentation for Bins Parameter when its supported
     def value_counts(self, normalize=False, sort=True, ascending=False, bins=None, dropna=True):
+        """
+        Return a Series containing counts of unique values.
+        The resulting object will be in descending order so that the
+        first element is the most frequently-occurring element.
+        Excludes NA values by default.
+        Parameters
+        ----------
+        normalize : boolean, default False
+            If True then the object returned will contain the relative
+            frequencies of the unique values.
+        sort : boolean, default True
+            Sort by values.
+        ascending : boolean, default False
+            Sort in ascending order.
+        bins : Not Yet Supported
+        dropna : boolean, default True
+            Don't include counts of NaN.
+        Returns
+        -------
+        counts : Series
+        See Also
+        --------
+        Series.count: Number of non-NA elements in a Series.
+        Examples
+        --------
+        >>> df = ks.DataFrame({'x':[0, 1, np.nan, 0, 2, np.nan, 3, 3]})
+        >>> df.x.value_counts()
+         3.0    2
+         0.0    2
+         1.0    1
+         2.0    1
```

Comment:
```
398         >>> df = ks.DataFrame({'x':[0, 1, np.nan, 0, 2, np.nan, 3, 3]})
399         >>> df.x.value_counts()
Differences (unified diff with -expected +actual):
    @@ -1,5 +1,5 @@
    - 3.0    2
    - 0.0    2
    - 1.0    1
    - 2.0    1
    +0.0    2
    +3.0    2
    +1.0    1
    +2.0    1
     Name: x, dtype: int64
```

Seems doctests are being failed due to spacing diff

Reply:
and the order of values

Reply:
You can remove the spaces and the order of expected result in the doctest.

Reply:
Did the changes. should be good to go.